packages/components/src/components/checkbox/checkbox.stories.ts

```diff
@@ -1,15 +1,18 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
 const { argTypes, parameters } = storybookDefaults('sd-checkbox');
 const { generateTemplate } = storybookTemplate('sd-checkbox');
 const { overrideArgs } = storybookHelpers('sd-checkbox');
 
+/**
+ * Checkboxes allow the user to toggle an option on or off.
```

Comment:
Allows users to toggle an option on or off.

Reply:
Updated